ops/terraform/modules/resources/prefix_list/main.tf

```diff
@@ -4,7 +4,7 @@
 // See https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/ec2_managed_prefix_list
 // for help on referencing an existing prefix list.
 
-// Provision the prefix list with optinal entry blocks.
```

Comment:
While I don't disagree with the correction, I don't believe it belongs _here_ 😉

Reply:
was on the fence about this, but you're right